app/segments/(team)/manageForeignRequests.js

```diff
@@ -0,0 +1,99 @@
+import { useNavigation } from "expo-router";
+import { useMemo, useState } from "react";
+import { Appbar, SegmentedButtons } from "react-native-paper";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { themeColors } from "~/Constants";
+import Blacklist from "~/app/segments/(team)/blacklist";
+import Header from "~/components/header";
+
+function ManageForeignRequests() {
+  const [tabValue, setTabValue] = useState("invites");
+  const navigation = useNavigation();
+
+  const segmentedColor = (tab) => {
+    switch (tab) {
+      case "invites":
+        return "#008001";
+      case "waitlist":
+        return "#FFE900";
+      case "blacklist":
+        return "#FE0100";
+      default:
+        return themeColors.overlay;
+    }
+  };
+
+  const segmentedTextColor = (tab) => {
+    switch (tab) {
+      case "invites":
+      case "blacklist":
+        return "white";
+      case "waitlist":
+        return "black";
+      default:
+        return "black";
+    }
+  };
+
+  const tabComponent = useMemo(
+    () => ({
+      invites: <></>,
+      waitlist: <></>,
+      blacklist: <Blacklist />,
+    }),
+    [],
+  ); // Recreate pages only if drillId changes
```

Comment:
Can you explain this comment a bit?

Reply:
I copied this from content/drill/[id] so it's a leftover comment